fishy/osservices/linux.py

```diff
@@ -1,29 +1,101 @@
+import subprocess
+import re
+import os
+from Xlib import *
```

Comment:
dont import everything, import the things u need to use... that way we can see which pakage is providing which variable

so change this to `from Xlib import display` if you aren't using anything else from the package